common/cftoFunctions.js

```diff
@@ -45,18 +48,44 @@ function chooseCollection() {
     browser.click('.collection--summary__btn');
 }
 
+function chooseCollectionIE() {
+    //Select store from the store list and confirm
+    browser.click('#store_4048');
```

Comment:
Sir, This ID is based on the Ireland Store ID. So, this ID is always going to be same for a respective store. So, i have used the ID of a "DUBLIN MARY ST" store which is 4048